App.js

```diff
@@ -2,7 +2,8 @@ import * as React from 'react';
 import DeviceInfo from 'react-native-device-info';
 import FlashMessage from 'react-native-flash-message';
 import Toast from 'react-native-toast-message';
-import {BackHandler, KeyboardAvoidingView, Platform, View} from 'react-native';
+import {BackHandler, Platform, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `KeyboardAvoidingView` import has been removed. If this was used to adjust the view when the keyboard is displayed, its removal might cause issues on screens where the keyboard is used. Please ensure that the removal of `KeyboardAvoidingView` does not affect the user experience on screens with input fields.
```diff
- import {BackHandler, KeyboardAvoidingView, Platform, View} from 'react-native';
+ import {BackHandler, Platform, View} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->